.eslintrc.js

```diff
@@ -3,6 +3,6 @@ module.exports = {
     extends: require.resolve('@diplodoc/lint/eslint-config'),
     parserOptions: {
         tsconfigRootDir: __dirname,
-        project: ['./tsconfig.json', './tsconfig.tests.json'],
```

Comment:
This code will be deleted every time by a pre-commit hook

Reply:
yes, I will change it in @diplodoc/lint package.

Reply:
In this PR, I went another way - I created an alternative TS configuration for the build.